orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -83,6 +83,10 @@ def __init__(self, module):
         # flag to enable symbolic shape inference for dynamic shape inputs to improve performance
         self._run_symbolic_shape_infer = True
 
+        # Use torch.onnx.OperatorExportTypes.ONNX_FALLTHROUGH to allow custom autograd.Functions'.
+        # Use torch.onnx.OperatorExportTypes.ONNX if pure ONNX is needed.
+        self._onnx_export_type = onnx_export_type
+
```

Comment:
Using `ONNX_FALLTHROUGH` is a more conservative choice. Since autograd.Function support is still in early stage, we might want to stay with 

Without `ONNX_FALLTHROUGH`, exporter may throw when seeing custom autograd.Function. The exporter part doesn't quite care shapes because it can access shapes as it still sees TorchIR.

Reply:
I believe we should include the exporter team to evaluate whether ONNX_FALLTHROUGH is safe.
My concern is that the exporter may not throw when unsupported ops. Although that might be desirable for the custom ops we care about, for the other ops we actually dont support, that might be an issue

Reply:
Switch back to ONNX successfully.